README.md

```diff
@@ -5,4 +5,78 @@
+|-------------------|--------------|------------------|------------------|-----------------|------------------|---------|--------------------|-------------------------|-----------------|----------------|
+| Identity provider | yes          | yes              | no               | yes             | yes              | yes              | no                | no                      | yes             | no             |
+
+## Resources
+
+For your convenience you can find the relevant standards linked below.
+
+- [Security Assertion Markup Language (SAML) V2.0 Technical Overview](http://docs.oasis-open.org/security/saml/Post2.0/sstc-saml-tech-overview-2.0.html)
+- [Assertions and Protocols for the OASIS Security Assertion Markup Language (SAML) V2.0 – Errata Composite](https://www.oasis-open.org/committees/download.php/35711/sstc-saml-core-errata-2.0-wd-06-diff.pdf)
+- [Bindings for the OASIS Security Assertion Markup Language (SAML) V2.0 – Errata Composite](https://www.oasis-open.org/committees/download.php/35387/sstc-saml-bindings-errata-2.0-wd-05-diff.pdf)
+- [Profiles for the OASIS Security Assertion Markup Language (SAML) V2.0 – Errata Composite](https://www.oasis-open.org/committees/download.php/35389/sstc-saml-profiles-errata-2.0-wd-06-diff.pdf)
+- [Metadata for the OASIS Security Assertion Markup Language (SAML) V2.0 – Errata Composite](https://www.oasis-open.org/committees/download.php/35391/sstc-saml-metadata-errata-2.0-wd-04-diff.pdf)
+- [Conformance Requirements for the OASIS Security Assertion Markup Language (SAML) V2.0 – Errata Composite](https://www.oasis-open.org/committees/download.php/35393/sstc-saml-conformance-errata-2.0-wd-04-diff.pdf)
+- [xml.com SAML Specifications](http://saml.xml.org/saml-specifications)
+- [SAML Developer Tools from Onelogin](https://www.samltool.com/online_tools.php)
+
+## Supported Go Versions
+
+For security reasons, we only support and recommend the use of one of the latest two Go versions (:white_check_mark:).  
+Versions that also build are marked with :warning:.
+
+| Version | Supported          |
+|---------|--------------------|
+| <1.15   | :x:                |
+| 1.15    | :warning:          |
+| 1.16    | :warning:          |
+| 1.17    | :white_check_mark: |
+| 1.18    | :white_check_mark: |
+| 1.19    | :white_check_mark: |
+
+## Why another library
+
+As of 2021 there are not a lot of `SAML` library's in `Go` which can handle server and client implementations. CAOS is
+strongly committed to the general field of IAM (Identity and Access Management) and as such, we need solid frameworks to
+implement services.
+
+There are some implementations around not specificly after the standard, so the library has to have the possibility to
+provide the functionality to
+still include the outliers.
```

Comment:
```suggestion
The existing libraries that we evaluated were not implementing the standard strictly. For that reason we developed this project to be compliant to the standard, while still having the possibility to handle outliers.
```

Reply:
@stebenz why don't we mention a few libraries that we had a look at? Could help others to evaluate a feasible implementation.

Reply:
added 4 of the libraries which were used as reference